prep-javascript-variables/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+  const fullName = 'Chase Simmonds';
+  console.log(fullName);
+
+  const numberOfPets = '1';
+  console.log(numberOfPets);
```

Comment:
Hi Chase, you'll want to leave out the quotes so that the JavaScript engine sees `numberOfPets` as a number and not a string.

Reply:
<img width="551" alt="Screen Shot 2022-03-01 at 5 01 10 PM" src="https://user-images.githubusercontent.com/65789518/156274688-da82f133-1849-4494-b78b-5403fca046af.png">